src/hitchhiker/tree/utils/cljs/async.cljc

```diff
@@ -72,6 +72,16 @@
        (recur (<? (go-f res f)) r)
        res))))
 
+(defn update-in< [m ks go-f & args]
```

Comment:
I think it would make sense to also port `map<` over? And potentially other common async helper functions we need in Datahike itself.

Reply:
There may be some more things we need. I was thinking we do another update once all the tests are passing in Datahike ClojureScript branch. As there may be one or two new functions that we also introduce in this namespace.

Reply:
Ok, sure.